slasher/db/min_max_span.go

```diff
@@ -40,7 +40,7 @@ func (db *Store) ValidatorSpansMap(validatorIdx uint64) (*ethpb.EpochSpanMap, er
 
 // SaveValidatorSpansMap accepts a validator index and span map and writes it to disk.
 func (db *Store) SaveValidatorSpansMap(validatorIdx uint64, spanMap *ethpb.EpochSpanMap) error {
-	err := db.update(func(tx *bolt.Tx) error {
+	err := db.batch(func(tx *bolt.Tx) error {
```

Comment:
Does this make a difference?

Reply:
yes. it makes a big difference as we have many go routines calling it . will confirm and put numbers

Reply:
Nice. Would love to see some flame graphs for this

Reply:
added graphs to description

Reply:
I dont have the older data. just changed from batch to update

Reply:
added benchmark data from master

Reply:
and for batch vs update